.ci/pitest-suppressions/pitest-api-suppressions.xml

```diff
@@ -18,15 +18,6 @@
     <lineContent>return Arrays.copyOf(fileExtensions, fileExtensions.length);</lineContent>
   </mutation>
 
-  <mutation unstable="false">
```

Comment:
Why did we need two tests in two different checks to kill one mutation?

Reply:
> kill one mutation?

There are **two** mutations, one for:
https://github.com/checkstyle/checkstyle/blob/696cd880e83b6c0af3171bf003ebb142eb4c0d35/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java#L213-L224

and the other for:
https://github.com/checkstyle/checkstyle/blob/696cd880e83b6c0af3171bf003ebb142eb4c0d35/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java#L226-L242

But its the limitation of the survival checking method that it cannot differentiate between the two.

Reply:
Please add this explanation to PR description.

Reply:
Done